lib/acl.js

```diff
@@ -510,7 +509,11 @@ function allow(mode, req, res, next) {
     var reqPath = res && res.locals && res.locals.path ? res.locals.path : req.path;
     var acl = reqToACL(req);
     acl.allow(mode, reqPath, function(err) {
-        next(err);
+        if (err) {
```

Comment:
Remove my comment you can just write next(err) if err is undefined, it passes to the next handler, no issues